tests/clients/python_client_test.py

```diff
@@ -24,8 +25,7 @@ async def test_editing_file_auto_accept(mock_call_llm_api, mock_setup_api_key):
         @@code
         # Line 2
         @@end""")])
-
-    python_client = PythonClient(paths=["."])
+    python_client = PythonClient(cwd=Path.cwd(), paths=["."])
```

Comment:
Need to specify `cwd` here since pathlib/os will get the correct `cwd` that pytest is working out of.